sky/cli.py

```diff
@@ -3013,6 +3013,7 @@ def _output():
                 'CLOUD',
                 'INSTANCE_TYPE',
                 'vCPUs',
+                'DEVICE_MEMORY',
                 'HOST_MEMORY',
```

Comment:
I think it's more natural to have `HOST_MEMORY` next to `vCPUs`?

Reply:
Great catch! Now using device_mem, vcpus, host_mem.